lib/improver/tests/percentile/test_PercentileConverter.py

```diff
@@ -145,6 +145,40 @@ def test_valid_multi_coord_string_list(self):
                               [[-180., 180.]])
         self.assertArrayEqual(result.coord('latitude').bounds, [[-90., 90.]])
 
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."])
+    def test_use_with_masked_data(self):
+        """
```

Comment:
I think in general, we've been trying to have our docstrings in unit tests follow the style in https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-testing, as depending how you run the unit tests, the first line of the docstring is also displayed. However, I can see in this case, this file all has a different style of docstring.